homeassistant/components/systemmonitor/sensor.py

```diff
@@ -52,71 +57,226 @@
-        None,
-        False,
+    SystemMonitorSensorEntityDescription(
+        key="disk_use",
+        name="Disk use",
+        unit_of_measurement=DATA_GIBIBYTES,
+        icon="mdi:harddisk",
+        device_class=None,
+        state_class=STATE_CLASS_MEASUREMENT,
+        mandatory=False,
     ),
-    "network_in": ("Network in", DATA_MEBIBYTES, "mdi:server-network", None, True),
-    "network_out": ("Network out", DATA_MEBIBYTES, "mdi:server-network", None, True),
-    "packets_in": ("Packets in", " ", "mdi:server-network", None, True),
-    "packets_out": ("Packets out", " ", "mdi:server-network", None, True),
-    "throughput_network_in": (
-        "Network throughput in",
-        DATA_RATE_MEGABYTES_PER_SECOND,
-        "mdi:server-network",
-        None,
-        True,
+    SystemMonitorSensorEntityDescription(
+        key="disk_use_percent",
+        name="Disk use (percent)",
+        unit_of_measurement=PERCENTAGE,
+        icon="mdi:harddisk",
+        device_class=None,
+        state_class=STATE_CLASS_MEASUREMENT,
+        mandatory=False,
     ),
-    "throughput_network_out": (
-        "Network throughput out",
-        DATA_RATE_MEGABYTES_PER_SECOND,
-        "mdi:server-network",
-        None,
-        True,
+    SystemMonitorSensorEntityDescription(
+        key="ipv4_address",
+        name="IPv4 address",
+        unit_of_measurement=None,
+        icon="mdi:server-network",
+        device_class=None,
+        state_class=None,
+        mandatory=True,
     ),
-    "process": ("Process", " ", CPU_ICON, None, True),
-    "processor_use": ("Processor use (percent)", PERCENTAGE, CPU_ICON, None, False),
-    "processor_temperature": (
-        "Processor temperature",
-        TEMP_CELSIUS,
-        None,
-        DEVICE_CLASS_TEMPERATURE,
-        False,
+    SystemMonitorSensorEntityDescription(
+        key="ipv6_address",
+        name="IPv6 address",
+        unit_of_measurement=None,
+        icon="mdi:server-network",
+        device_class=None,
+        state_class=None,
+        mandatory=True,
     ),
-    "swap_free": ("Swap free", DATA_MEBIBYTES, "mdi:harddisk", None, False),
-    "swap_use": ("Swap use", DATA_MEBIBYTES, "mdi:harddisk", None, False),
-    "swap_use_percent": ("Swap use (percent)", PERCENTAGE, "mdi:harddisk", None, False),
-}
+    SystemMonitorSensorEntityDescription(
+        key="last_boot",
+        name="Last boot",
+        unit_of_measurement=None,
+        icon=None,
+        device_class=DEVICE_CLASS_TIMESTAMP,
+        state_class=None,
+        mandatory=False,
+    ),
+    SystemMonitorSensorEntityDescription(
+        key="load_15m",
+        name="Load (15m)",
+        unit_of_measurement=None,
+        icon=CPU_ICON,
+        device_class=None,
+        state_class=STATE_CLASS_MEASUREMENT,
+        mandatory=False,
+    ),
+    SystemMonitorSensorEntityDescription(
+        key="load_1m",
+        name="Load (1m)",
+        unit_of_measurement=None,
+        icon=CPU_ICON,
```

Comment:
You are mixing variables and strings for the icon, stay consistent

Reply:
only `CPU_ICON` is used, since it is set based on system if it is 32 or 64bit system (see line 56)

```python
if sys.maxsize > 2 ** 32:
    CPU_ICON = "mdi:cpu-64-bit"
else:
    CPU_ICON = "mdi:cpu-32-bit"
```